src/enclave/claims.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
`enclave/` seems like an odd directory for this new file. Maybe `node/rpc` instead?

Reply:
Yeah I don't remember how it landed there. I think we want the header under include/ccf in fact, and the cpp somewhere else maybe node/rpc.